Lesson7v3/auth.py

```diff
@@ -0,0 +1,74 @@
+import json
+
+with open("db.json", "r") as f_json:
+    data = json.load(f_json)
+
+
+def check_name(data):
+    """Here we are getting a nickname for a new users registration and checking it for individuality"""
+    active = True
+    while active:
+        username = input('Enter your nickname for a registration.' "\n")
```

Comment:
better to divide this to another function like .. `get_name` for example..
because I expect `check_name` will pass name and check this name..